internal/driver/interactive.go

```diff
@@ -42,6 +42,16 @@ func interactive(p *profile.Profile, o *plugin.Options) error {
 	interactiveMode = true
 	shortcuts := profileShortcuts(p)
 
+	// Get all groups in pprofVariables to allow for clearer error messages.
+	groups := make(map[string][]string)
+	for name, option := range pprofVariables {
+		group := option.group
+		if group != "" {
+			groupValues := groups[group]
```

Comment:
I think you can get rid of this local variable, it's only used once below.

Reply:
Done.